lib/sidekiq/client.rb

```diff
@@ -186,8 +186,8 @@ def enqueue_in(interval, klass, *args)
 
     def raw_push(payloads)
       @redis_pool.with do |conn|
-        conn.multi do
-          atomic_push(conn, payloads)
+        conn.pipelined do |pipeline|
```

Comment:
I assume this is safe to change from a `multi` to a `pipeline` b/c [the latest sidekiq does the same](https://github.com/sidekiq/sidekiq/blob/145601d4cbcab03f999ab54e75d884e7411a3bd3/lib/sidekiq/client.rb#L224-L230), but what is the reason for this?

Reply:
the reason is linked in this PR's description, https://github.com/sidekiq/sidekiq/pull/5043

also, some existing platform code is causing this to be a nested `multi`, which now raises an error